### PR TITLE
Testing part2

### PR DESCRIPTION
**Pull request by yashodhicy**
this pull request is to merge the changes due to adding the following.

1. Use jest framework for testing.
2. Create a test file(s) ([..].test.js) for a file(s) containing the following functions:
3. a function for editing the task description.
4. a function for updating an item's 'completed' status.
5. the "Clear all completed" function.
6. Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
7. Mock the HTML elements to test DOM manipulation functions.
8. fix linters

### CODE REVIEW

File: src/completedAll.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    { description: 'Task 1', completed: true },
+    { description: 'Task 2', completed: false },
+    { description: 'Task 3', completed: true },
+  ],
+  taskPopulate: jest.fn(),
+  setStorage: jest.fn(),
+}));
+
+describe('removeCompletedTask', () => {
+  test('Removes completed tasks', () => {
```
Comment (afanimartin) [inline src/completedAll.test.js]: - Please add a test for the function for editing an item's description
- Add another test for a function for updating an items 'completed' status
- Also, test the "Clear All Completed" function

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- You created a test file and added a test for a function to remove completed item. Well done
- All participants of the team contributed to the PR. Nice work

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/completedAll.test.js (hunk 1 above)
Comment (yashodhicy) [reply src/completedAll.test.js]: hi @afanimartin , we have added those files in modules.
image

**Review by Shisui6 — APPROVED** [review]
Hi @yashodhicy, @binodbhusal and @Jessekay, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- All tests pass ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**